src/audio/google/google_rtc_audio_processing.c

```diff
@@ -424,15 +429,23 @@ static struct comp_dev *google_rtc_audio_processing_create(
 		goto fail;
 	bzero(cd->raw_mic_buffer, cd->num_frames * cd->num_capture_channels * sizeof(cd->raw_mic_buffer[0]));
 	cd->raw_mic_buffer_frame_index = 0;
+	for (channel = 0; channel < cd->num_capture_channels; channel++)
+		cd->raw_mic_buffer_ptrs[channel] = &cd->raw_mic_buffer[channel * cd->num_frames];
```

Comment:
Looks like there's a `!aec_reference_buffer` early exit case just below this, probably best to put this after that out of general "do the cheap checks first" hygiene.  It also allows the compiler (maybe) or you to roll this into the other num_capture_channels loop you're adding below.  I don't think I see any other branching here that would prevent that, though it's hard to tell in a review UI.